README.md

```diff
@@ -1,15 +1,9 @@
-#Gentoo Science Overlay
+#Fork of Gentoo Science Overlay
 
-An overlay for Gentoo, geared towards scientific packages.
+This a fork to add and maintain scientific packages in Gentoo Science Overlay. 
 
-Homepage: [Science Project page](https://wiki.gentoo.org/wiki/Project:Science/Overlay)
-
-
-Please fork! We will merge! See [this](https://github.com/gentoo-science/sci/blob/master/CONTRIBUTING.md) document for more instructions.
 
-Ask for help on irc in #gentoo-science @ freenode.
-
-Report bugs on the [github issues site](https://github.com/gentoo-science/sci/issues)
+Homepage: [Science Project page](https://wiki.gentoo.org/wiki/Project:Science/Overlay)
 
 
-[![Build Status](https://travis-ci.org/gentoo-science/sci.svg?branch=master)](https://travis-ci.org/gentoo-science/sci)
+[![Build Status](https://travis-ci.org/freeh4cker/sci.svg?branch=master)](https://travis-ci.org/freeh4cker/sci)
```

Comment:
Ok,

you should notice that in CONTRIBUTING, because I work with some other
projects like biopython and they ask to change  the readme to dont confuse
people with the official repository and clones.

I have a question. If I keep the readme as in the original, the travis-ci
flag on my clone will point to gentoo-science official travis-ci tests, so
even my repository is broken the flag will be green  it's weird insn't it?

bertrand

On Mon, Apr 27, 2015 at 2:03 PM, marbre <notifications@github.com> wrote:

> In README.md
> <https://github.com/gentoo-science/sci/pull/411#discussion_r29140209>:
>
> >
> >
> > -[![Build Status](https://travis-ci.org/gentoo-science/sci.svg?branch=master)](https://travis-ci.org/gentoo-science/sci)
> > +[![Build Status](https://travis-ci.org/freeh4cker/sci.svg?branch=master)](https://travis-ci.org/freeh4cker/sci)
>
> Please don't edit the README.md.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gentoo-science/sci/pull/411/files#r29140209>.
>